config.js

```diff
@@ -10,6 +10,9 @@ const { getOptionName } = require('./utils/getOptionName');
 const { getRankedUnranked } = require('./utils/getRankedUnranked');
 const { getRankedUnrankedUnderscored } = require('./utils/getRankedUnrankedUnderscored');
 
+const ogsPvAIs = require('./constants').constants.ogsPvAIs;
+const rankedUnrankedOptions = require('./constants').constants.rankedUnrankedOptions;
```

Comment:
Can we not do

```Js
const { ogsPvAIs, rankedUnrankedOptions } = require('./constants').constants
```

Also. Can we not remove `constants` in there and make it use `require('./constants')` directly?

Reply:
yes, that would remove a repetition :1st_place_medal: 

i rebased on top of latest devel, and also moved droppedOptions to constants.js :100:

Reply:
tested to work on beta ogs

Reply:
tested to work on beta ogs :+1: